OmniGibson/omnigibson/learning/utils/network_utils.py

```diff
@@ -47,7 +50,26 @@ def get_server_metadata(self) -> Dict:
         return self._server_metadata
 
     def _wait_for_server(self) -> Tuple[websockets.sync.client.ClientConnection, Dict]:
-        logger.info(f"Waiting for server at {self._uri}...")
+        # Extract host and port for health check
+        host_port = self._uri.replace("ws://", "").replace("wss://", "")
+        if ":" in host_port:
+            host, port = host_port.split(":")
+            health_url = f"http://{host}:{port}/healthz"
+        else:
+            health_url = f"http://{host_port}/healthz"
```

Comment:
nit: things like this should be done with a URL parser instead of this. but I think we'll survive for now.